staging/publishing/rules.yaml

```diff
@@ -24,17 +24,17 @@ rules:
       branch: release-1.21
       dir: staging/src/k8s.io/code-generator
     name: release-1.21
-    go: 1.16.12
+    go: 1.16.13
```

Comment:
usually, we bump the go for the release branches after all PRs from k/k for the go updates are merged
in this PR we just update the `default-go-version`

Reply:
Ack! Will remove the branch rule updates.

Is it to maintain some kind of chronology of changes?

Reply:
PS: Removed the branch rule updates.

Reply:
to avoid to publish bot runs without having the correct versions I guess